### PR TITLE
[PATCH v3] github_ci: change default test image to ubuntu 20.04

### PR DESCRIPTION
**Pull request by MatiasElo**

### CODE REVIEW

File: .github/workflows/ci-pipeline.yml [hunk 1]
```diff
@@ -134,32 +135,31 @@ jobs:
...
-               -e CONF="${CONF}" $CONTAINER_NAMESPACE/odp-ci-${OS}-${ARCH} /odp/scripts/ci/build_${ARCH}.sh
+               -e CONF="${CONF} ${CONF_EXTRA}" $CONTAINER_NAMESPACE/odp-ci-${OS}-${ARCH} /odp/scripts/ci/build_${ARCH}.sh
 
-      - name: Ubuntu 20.04
+      - name: Ubuntu 18.04
         env:
-          CONF: "--enable-dpdk-shared"
-          OS: ubuntu_20.04
+          OS: ubuntu_18.04
         run: sudo docker run -i -v `pwd`:/odp --privileged --shm-size 8g -e CC="${{matrix.compiler}}"
```
Comment (psavol) [inline .github/workflows/ci-pipeline.yml:163]: No such matrix: matrix.compiler

File: .github/workflows/ci-pipeline.yml [hunk 2]
```diff
@@ -251,23 +252,6 @@ jobs:
         if: ${{ failure() }}
         run: find . -name config.log -exec cat {} \;
 
-  Build_gcc_u20:
```
Comment (psavol) [inline .github/workflows/ci-pipeline.yml:254]: Could be changed to ubuntu 18 build. Test cases for GCC versions 7 and 8.

File: .github/workflows/ci-pipeline.yml [hunk 3]
```diff
@@ -134,32 +135,31 @@ jobs:
...
       - name: "--with-crypto=armv8crypto"
         env:
-          CONF: "--with-crypto=armv8crypto"
+          CONF_EXTRA: "--with-crypto=armv8crypto"
         run: sudo docker run -i -v `pwd`:/odp --privileged --shm-size 8g -e CC="${{matrix.cc}}"
-               -e CONF="${CONF}" $CONTAINER_NAMESPACE/odp-ci-${OS}-${ARCH} /odp/scripts/ci/build_${ARCH}.sh
+               -e CONF="${CONF} ${CONF_EXTRA}" $CONTAINER_NAMESPACE/odp-ci-${OS}-${ARCH} /odp/scripts/ci/build_${ARCH}.sh
 
-      - name: Ubuntu 20.04
+      - name: Ubuntu 18.04
```
[the remark this answers is not in the document]
Comment (MatiasElo) [reply .github/workflows/ci-pipeline.yml]: Done in V2.

File: .github/workflows/ci-pipeline.yml (hunk 2 above)
Comment (MatiasElo) [reply .github/workflows/ci-pipeline.yml:254]: Done in V2.

**Pull request merged**